### PR TITLE
Admin backend

### PR DESCRIPTION
**Pull request by shubham1172**
Pull for issue #3 

- [x] Added admin backend
- [x] Added admin unit tests
- [x] Added auth patch

Awaiting test cases from @the-lost-explorer.

### CODE REVIEW

**Comment by shubham1172** [comment]
@the-lost-explorer, the new User schema broke the old build's test cases. Can you update the `before` function in `test/api/auth.test.js` and fix the problem? 

Logs
```
1) "before all" hook: inserting dummy data for testing:
     User validation failed: profile_picture_url: Path `profile_picture_url` is required., password: Path `password` is required.
  ValidationError: User validation failed
      at ValidationError (node_modules/mongoose/lib/error/validation.js:30:11)
      at model.Document.invalidate (node_modules/mongoose/lib/document.js:2085:32)
      at p.doValidate.skipSchemaValidators (node_modules/mongoose/lib/document.js:1948:17)
      at /home/circleci/project/node_modules/mongoose/lib/schematype.js:929:9
      at _combinedTickCallback (internal/process/next_tick.js:73:7)
      at process._tickCallback (internal/process/next_tick.js:104:9)
```

**Comment by the-lost-explorer** [comment]
> @the-lost-explorer, the new User schema broke the old build's test cases. Can you update the `before` function in `test/api/auth.test.js` and fix the problem?
> 
> Logs
> 
> ```
> 1) "before all" hook: inserting dummy data for testing:
>      User validation failed: profile_picture_url: Path `profile_picture_url` is required., password: Path `password` is required.
>   ValidationError: User validation failed
>       at ValidationError (node_modules/mongoose/lib/error/validation.js:30:11)
>       at model.Document.invalidate (node_modules/mongoose/lib/document.js:2085:32)
>       at p.doValidate.skipSchemaValidators (node_modules/mongoose/lib/document.js:1948:17)
>       at /home/circleci/project/node_modules/mongoose/lib/schematype.js:929:9
>       at _combinedTickCallback (internal/process/next_tick.js:73:7)
>       at process._tickCallback (internal/process/next_tick.js:104:9)
> ```

Sure.

File: utils/base64.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+exports.decode_image = (data_string) => {
```
Comment (the-lost-explorer) [inline utils/base64.js]: I used image-to-base64 library. How is this better?

Comment (shubham1172) [reply utils/base64.js]: It's just written in the house and there are a few particular specifications!

**Pull request merged**